core/templates/pages/exploration-player-page/templates/lesson-information-card-modal.component.html

```diff
@@ -125,6 +143,36 @@ <h4 class="oppia-exploration-description" *ngIf="isHackyExpDescTranslationDispla
   <i class="material-icons md-18">&#xE5CD;</i>
   <span class="oppia-icon-accessibility-label">Close</span>
 </button>
+<div class="save-progress-menu"
+     *ngIf="saveProgressMenuIsShown && !userIsLoggedIn">
+  <h3>Save Progress</h3>
+  <div class="sign-in-box">
+    <p>Your learning progress will be automatically saved if you have an account.</p>
+    <button type="button"
+            class="btn btn-secondary create-account-btn"
+            (click)="onLoginButtonClicked()">
+          Create Account
+    </button>
+    <p class="sign-in-text">Already have an account? <a class="sign-in-link" (click)="onLoginButtonClicked()">Sign In</a></p>
+  </div>
+  <div class="separator">
+    <div class="bars"></div>
+    <div class="mid">or</div>
+    <div class="bars"></div>
+  </div>
+  <div class="uid-box">
+    <p class="guide-text">Use the link below to save progress for 72 hours.<br>You may lose your progress after 72 hours.</p>
+    <p class="copy-text">Write or copy the link below</p>
+    <div class="uid-copy" [ngStyle]="{ 'flex-direction': isLanguageRTL() ? 'row-reverse' : 'row' }">
+      <div id="unique-progress-id">{{ loggedOutProgressUniqueUrl }}</div>
```

Comment:
Why do we need to use the ID attribute here?

Reply:
Updated it now.